pom.xml

```diff
@@ -77,4 +75,41 @@
 		</repository>
 	</repositories>
 
+	<build>
```

Comment:
Eclipse claims an error with lifecycle configuration. In fix offers it generated this part. As the DPU works without it as well i actually revomed it.